### ISSUE TITLE
Why display: table instead of Flexbox?

### ISSUE THREAD

**Issue by afontcu** [inuitcss/inuitcss#301]
Hi there, I've been playing around with Inuit for a while now and I'm wondering why some objects such as Flag or Pack are made using `display: table` or similar instead of using Flexbox. Is it due to IE bugs? Am I missing something? Could we update this objects and make them Flex? I'd say then Media and Flag would become the same object (since there's only a vertical alignment difference between the two).

The base Media object would be really simple (as it is now):

```sass
.o-media {
    display: flex;
    align-items: flex-start;
}

.o-media__img {
    margin-right: $inuit-global-spacing-unit;
}

.o-media__body {
    flex: 1;

    &,
    > :last-child {
        margin-bottom: 0;
    }
}
```

(some tweaks could be added obviously).

Thanks!

**Comment by pnowak2** [comment]
im pretty sure its about vertical alignment which works reliably with
display: table ?
Correct me if im wrong, but this is usually the case, to have better
vertical align control and avoiding line dropping/wrapping.

czw., 11 maj 2017 o 15:58 użytkownik afontcu <notifications@github.com>
napisał:

**Comment by florianbouvot** [comment]
@afontcu We are aware that flexbox is a good solution but for now it's about browser compatibility (we want to support old IE).

We will maybe propose flexbox objects as official plugins (/cc @inuitcss/core).

**Comment by afontcu** [comment]
@pnowak2 I'd say you can achieve the same vertical alignment reliability with `table` or `flex`.

@florianbouvot makes sense, just wanted to confirm that. You can close this if you want.

Thank you both!

**Comment by florianbouvot** [comment]
@afontcu without re-launching a major debate, what browser do you support?

**Comment by afontcu** [comment]
@florianbouvot IE10, so 94.5% of usage in Spain which bumps to 98,5% using prefixes (according to caniuse.com). 

Yes, I need to specify `flex: 0 1 auto` as default since IE sets `0 0 auto` and things like that, but IMHO the benefits outweigh the caveats of a few (documented) bugs.

Things got easier once I discovered [Flexbugs](https://github.com/philipwalton/flexbugs).

Is Inuit supporting IE8 and IE9? Found no references in the repo.
